Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM node:18 AS vendors
+
+COPY . /srv/umap
+
+WORKDIR /srv/umap
+
+RUN make installjs
+
+RUN make vendors
+
+FROM python:3.8-slim
```

Comment:
Maybe 3.11 now ?

Reply:
I'm not a Python programmer, so if that is the latest we can use that :)